app/components/school_information/your_schools/participants_component.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="govuk-grid-row govuk-!-margin-bottom-3">
+  <div class="govuk-grid-column-one-half">
+    <div class="dashboard-numbers key-number">
+      <p class="govuk-heading-xl">300</p>
```

Comment:
Ditto, ticket to make it dynamic?

Reply:
Lead provider journey is stll being spec out and changed by Al and Shani - so for now we don't have the details how this will change or if it will exist at all